client/packages/system/src/Vaccination/Components/getSwitchReason.test.ts

```diff
@@ -5,52 +5,51 @@ import { OTHER_FACILITY } from './FacilitySearchInput';
 
 describe('getSwitchReason', () => {
   describe('no existing vaccination (create)', () => {
-    it('should return "record transaction" if the date is historical & doses configured', () => {
+    it('should return "record transaction" if the date is historical & item selected', () => {
```

Comment:
test changes are just noise due to usage change - instead of there being no dose items configured, should be if none is selected.